spec/lib/ability_spec.rb

```diff
@@ -143,4 +143,15 @@
       expect(subject.can?(:manage, SystemNote.new)).to eq true
     end
   end
+
+  context "as a client support user" do
+    let!(:user) { create :user, is_client_support: true }
+    let(:client_1) { create(:client, vita_partner: create(:vita_partner)) }
+    let(:client_2) { create(:client, vita_partner: create(:vita_partner)) }
+
+    it "can see all the clients from all the organizations" do
```

Comment:
I would like to see a negative test here validating that they can't `:manage` clients. It doesn't have to block this pull request.

I would also like to avoid the use of `:all` in the Ability.rb -- I think it could confuse us into creating excessively-broad permissions.

Reply:
@alaroia-cfa good points! I added a test to check that client support users could not manage clients and changed `:all` in `Ability.rb` to `[Clients]`